src/psyclone/transformations.py

```diff
@@ -2563,7 +2562,10 @@ def apply(self, node, options=None):
         for child in routine.children:
             if isinstance(child, ACCRoutineDirective):
                 return  # The routine is already marked with ACCRoutine
-        routine.children.insert(0, ACCRoutineDirective())
+
+        para = options.get("parallelism", None) if options else None
```

Comment:
None -> "seq" if you do the change above

Reply:
Done.